deker_server_adapters/factory.py

```diff
@@ -125,12 +124,18 @@ def get_collection_adapter(
         """
         return ServerCollectionAdapter(self.ctx)
 
-    def do_healthcheck(self, ctx: CTX, in_cluster: bool = False) -> Optional[Dict]:
+    def __uri_has_servers(self, ctx: CTX) -> bool:
+        """Check if URI has servers.
+
+        :param ctx: Context with URI
+        """
+        return hasattr(ctx.uri, "servers") and ctx.uri.servers
```

Comment:
Do we really need this method? Can't we just check `if ctx.uri.servers`?  
As I see, deker.uri always has this attribute (may be we shall refactor it a bit).

Reply:
I'm not sure tbh that `deker.uri` would have `servers` attribute always, so it's just a precaution. Might be refactored later.

Reply:
Yes, it does.
![image](https://github.com/openweathermap/deker-server-adapters/assets/64586748/7be93897-9052-4e7c-8a05-b74523fc3a9c)

Reply:
https://github.com/openweathermap/deker-server-adapters/pull/23/commits/dff1e28bf33a2f7e21d2ff8aef5e4ed161e36c21